trino-docs.sh

```diff
@@ -42,71 +42,15 @@ find -H "$VERSION" -type f -name '*.html' -print0 | xargs -0 perl -pi -e \
       window._linkedin_data_partner_ids = ['$LINKEDIN_ID'];
     </script>
     <script async src=\"https://snap.licdn.com/li.lms-analytics/insight.min.js\"></script>
-    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.8.2/css/all.css">
```

Comment:
fa is not used needed anywhere else in the docs and if it will in the future we will incorporate directly in sphinx setup rather than hack to it later like this